test-framework/common/src/main/java/io/quarkus/test/common/NativeImageLauncher.java

```diff
@@ -130,16 +130,10 @@ private static String guessPath(Class<?> testClass) {
     }
 
     private static void logGuessedPath(String guessedPath) {
-        String errorString = "\n=native.image.path was not set, making a guess that  " + guessedPath
-                + " is the correct native image=";
-        for (int i = 0; i < errorString.length(); ++i) {
-            System.err.print("=");
-        }
-        System.err.println(errorString);
-        for (int i = 0; i < errorString.length(); ++i) {
-            System.err.print("=");
-        }
-        System.err.println();
+        System.err.println("======================================================================================");
+        System.err.println("  native.image.path was not set, making a guess for the correct path of native image");
+        System.err.println("  guessed path: " + guessedPath);
+        System.err.println("======================================================================================");
```

Comment:
It might be a stupid question but we can't use a proper logger here?

I mean a simple WARN would have done it? But maybe it's too early, not practical, whatever.

Reply:
I think there is no logging used in common module of test-framework.

I've just adjusted approach used before, NativeImageLauncher uses System.out in other places too.

Reply:
OK, thanks, I merged it.

Reply:
Thanks